solutions/telemetry/quarkus/consumer/src/main/java/com/redhat/telemetry/TemperaturesStreamResource.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.telemetry;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.reactivestreams.Publisher;
+
+@Path("/temperatures")
+public class TemperaturesStreamResource {
+
+    @Inject
+    TemperaturesConsumer consumer;
+
+    @GET
+    @Produces(MediaType.SERVER_SENT_EVENTS)
+    public Publisher<String> getTemperaturesStream() {
```

Comment:
Why is a Publisher when we are on a consumer?

Reply:
It found this to be a way to publish server sent events to the "frontend" (or to the endpoint caller)

Reply:
Ok, I changed this to `Multi`